listen.js

```diff
@@ -0,0 +1,6 @@
+const app = require("./app");
```

Comment:
There is no requirement to have a listen.js just yet. 
Supertest is doing this for you whilst we're testing. Needing our own listen.js will come later in the sprint